README.md

```diff
@@ -16,18 +16,133 @@ There are two components in this repository:
+| JAX        | ResNet50                            | FP32, FP16, BF16                      | 1, 8, 64, 128, 256, 2048               | Input image 3x224x224                    |
+| JAX        | GPT-2 with LMHead                   | FP32                                  | 1                                      | Sequence length 512                      |
+| JAX        | ViT                                 | FP32                                  | 1                                      | Input image 3x224x224                    |
+| PyTorch    | Bert-Large                          | FP32, FP16                            | 1, 16, 24, 32, 48, 64, 512, 1024, 1280 | Sequence length 384                      |
+| PyTorch    | ResNet50                            | FP32, FP16                            | 1, 8, 64, 128, 256, 2048               | Input image 3x224x224                    |
+| Tensorflow | T5-Large                            | FP32                                  | 1, 16, 24, 32, 48, 64, 512             | Input sequence 512                       |
+| Tensorflow | Bert-Large                          | FP32                                  | 1, 16, 24, 32, 48, 64, 512, 1024, 1280 | Input sequence 384                       |
+| Tensorflow | RestNet50                           | FP32                                  | 1, 8, 64, 128, 256, 2048               | Input image 224x224x3                    |
+| Tensorflow | EfficientNet-B7                     | FP32                                  | 1, 64, 128                             | Input image 600x600x3                    |
+| TFLite     | Bert-Base                           | FP32, FP16, Dynamic-range quant, INT8 | 1                                      | Input sequences 8, 32, 64, 128, 256, 512 |
+| TFLite     | ViT                                 | FP32, FP16, Dynamic-range quant, INT8 | 1                                      | Input image 3x224x224                    |
+
+### Pipeline
+
+Pipelines may include more than one model or control flow.
+
+| Framework  | Pipeline          | Data Type        | Variations                                   |
+|------------|-------------------|------------------|----------------------------------------------|
+| JAX        | T5-Small          | FP32, FP16, BF16 | Token generation sizes: 16, 32, 64, 128, 256 |
+| JAX        | Stable Diffusion  | FP32, FP16, BF16 | Input sequence 64 tokens                     |
+| JAX        | GPT-2 with LMHead | FP32             | Generates 200 tokens                         |
+| Tensorflow | GPT-2 with LMHead | FP32             | Generates 200 tokens                         |
+| GGML       | GPT-2 with LMHead | FP32, FP16       | Generates 200 tokens                         |
+
+## Dashboards
+
+* (Retired) IREE vs XLA and Inductor:
+[Looker Studio dashboard.](https://lookerstudio.google.com/c/u/0/reporting/e1a3146d-ae02-4dad-9501-11524cc4cea3/page/p_lzy2eg805c)
```

Comment:
I think this dashboard is not publicly accessible. And we might want to check with XLA team before publishing it

Reply:
Ohh yeah it needs to be shared with people. I'll remove it for now.